feature-governance-impl/src/main/java/io/novafoundation/nova/feature_governance_impl/presentation/tindergov/cards/TinderGovCardsViewModel.kt

```diff
@@ -123,14 +143,40 @@ class TinderGovCardsViewModel(
         }
     }
 
+    private fun loadFirstCards() {
```

Comment:
Can you explain why this is needed? Doesnt `onCardAppeared` for the first card will trigger the same?

Reply:
I tested it few times and onCardAppeared isn't called always for first time